aat/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.3-all.zip
```

Comment:
this version is ancient, why would you use this on something new =/

Reply:
You're right, but gradle upgrades will be taken care of as a separate activity. I just don't want to delay this pending such upgrade.